src/JsonLd/ContextBuilder.php

```diff
@@ -112,6 +112,7 @@ public function getAnonymousResourceContext(object $object, array $context = [],
         $outputClass = $this->getObjectClass($object);
         $operation = $context['operation'] ?? new Get(shortName: (new \ReflectionClass($outputClass))->getShortName());
         $shortName = $operation->getShortName();
+        $isAliasIdType = $operation->getAliasIdType();
```

Comment:
I'd like to find a better name for this, like `jsonldAsJson` or `jsonAliases`, or we need to find something ^^

Reply:
I don't like the current name too, some suggestions: `noSpecialProps`, `cleanOutput`

Reply:
Indeed, I'm not a fan of the name either. Perhaps `aliasJsonLdKeywords` or `keywordsAsJson` would be more appropriate? As noted in the [specification](https://www.w3.org/TR/json-ld11/#keywords), the terms `@id` / `@type` are referred to as "keywords".

Reply:
I don't like "keywords" because they are other keywords than `@id` and `@type`. As this options is closely related to the existing `jsonLdContext` property (https://api-platform.com/docs/core/extending-jsonld-context/), what about `jsonLdAliases: true`?

Reply:
`JsonLdAliases` works well for me! It's much more descriptive than `AliasIdType`.